prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -0,0 +1,336 @@
+package gov.cdc.prime.router.azure
```

Comment:
This is the main new class supporting lineage tracking.    Documentation in the code, I hope.